From a846f5f62a8b5f7e1558bfc1805c898b7e4ec5fe Mon Sep 17 00:00:00 2001
From: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Date: Tue, 21 Jul 2026 01:39:46 +0000
Subject: [PATCH 06272/15333] ASoC: qcom: common: use dev

qcom_snd_parse_of() already have dev. Let's use it.
No functional change, but is preparation for Card capsuling.

Signed-off-by: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Reviewed-by: Srinivas Kandagatla <srinivas.kandagatla@oss.qualcomm.com>
Link: https://patch.msgid.link/87pl0hi0il.wl-kuninori.morimoto.gx@renesas.com
Signed-off-by: Mark Brown <broonie@kernel.org>
---
 sound/soc/qcom/common.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/sound/soc/qcom/common.c b/sound/soc/qcom/common.c
index edc4611691f7..e4ff247ea47f 100644
--- a/sound/soc/qcom/common.c
+++ b/sound/soc/qcom/common.c
@@ -91,7 +91,7 @@ int qcom_snd_parse_of(struct snd_soc_card *card)
 
 		ret = of_property_read_string(np, "link-name", &link->name);
 		if (ret) {
-			dev_err(card->dev, "error getting codec dai_link name\n");
+			dev_err(dev, "error getting codec dai_link name\n");
 			return ret;
 		}
 
@@ -109,7 +109,7 @@ int qcom_snd_parse_of(struct snd_soc_card *card)
 
 		ret = snd_soc_of_get_dlc(cpu, &args, link->cpus, 0);
 		if (ret) {
-			dev_err_probe(card->dev, ret,
+			dev_err_probe(dev, ret,
 				      "%s: error getting cpu dai name\n", link->name);
 			return ret;
 		}
@@ -126,7 +126,7 @@ int qcom_snd_parse_of(struct snd_soc_card *card)
 					"sound-dai",
 					0);
 			if (!link->platforms->of_node) {
-				dev_err(card->dev, "%s: platform dai not found\n", link->name);
+				dev_err(dev, "%s: platform dai not found\n", link->name);
 				return -EINVAL;
 			}
 		} else {
@@ -136,7 +136,7 @@ int qcom_snd_parse_of(struct snd_soc_card *card)
 		if (codec) {
 			ret = snd_soc_of_get_dai_link_codecs(dev, codec, link);
 			if (ret < 0) {
-				dev_err_probe(card->dev, ret,
+				dev_err_probe(dev, ret,
 					      "%s: codec dai not found\n", link->name);
 				return ret;
 			}
-- 
2.55.0

